src/os/bluestore/BlueStore.cc

```diff
@@ -5353,12 +5403,13 @@ int BlueStore::_balance_bluefs_freespace(PExtentVector *extents)
 
     if (alloc_len <= 0) {
       dout(0) << __func__ << " no allocate on 0x" << std::hex << gift
-              << " min_alloc_size 0x" << min_alloc_size << std::dec << dendl;
+              << " min_alloc_size 0x" << cct->_conf->bluefs_alloc_size
+	      << std::dec << dendl;
       _dump_alloc_on_rebalance_failure();
       return 0;
     } else if (alloc_len < (int64_t)gift) {
       dout(0) << __func__ << " insufficient allocate on 0x" << std::hex << gift
-              << " min_alloc_size 0x" << min_alloc_size 
+              << " min_alloc_size 0x" << cct->_conf->bluefs_alloc_size
```

Comment:
why the switch here? this seems dangerous because the config (which only affects mkfs) may disagree with the ondisk store's min_alloc_size

Reply:
This is to better reflect previous allocation failure. It uses cct->_conf->bluefs_alloc_size not min_alloc_size. May be rename the caption?